woocommerce/woocommerce-android
# Track ORDERS_LIST_LOADED

Closes #3568. FluxC changes were already done and merged: https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/1882. 

This adds tracking for how long `GET /orders` requests happen for our customers. This data is only one of the data points we can use to inform our progress on the Slow Orders project (p91TBi-4p5-p2).

## Testing

2. Log in and navigate to the Orders list → Processing. Pull to refresh.
3. Confirm that there is a log like this:
    ```
    🔵 Tracked: orders_list_loaded, Properties: {"total_duration":1.148,"status":"processing","blog_id":171751943,"is_wpcom_store":false}
    ```
4. Navigate to All Orders. Pull to refresh.
5. Confirm that there is a log like this:
    ```
    🔵 Tracked: orders_list_loaded, Properties: {"total_duration":1.148,"status":null,"blog_id":171751943,"is_wpcom_store":false}
    ```
5. Scroll down so the next set of Orders is loaded. Make sure you have lots of orders though. 😅 
5. Confirm that there is a log like this:
    ```
    🔵 Tracked: orders_list_loaded, Properties: {"total_duration":1.148,"status":null,"blog_id":171751943,"is_wpcom_store":false}
    ```
6. Tap on Search and pick the Completed status. 
5. Confirm that there is a log like this:
    ```
    🔵 Tracked: orders_list_loaded, Properties: {"total_duration":1.148,"status":"completed","blog_id":171751943,"is_wpcom_store":false}
    ```

## Author Checklist
